frb_codegen/src/generator/dart/ty.rs

```diff
@@ -21,6 +21,15 @@ pub trait TypeDartGeneratorTrait {
 #[derive(Debug, Clone)]
 pub struct TypeGeneratorContext<'a> {
     pub ir_file: &'a IrFile,
+    pub config: &'a Opts,
+    pub dart_api_class_name: Option<String>,
```

Comment:
nit: Is `dart_api_class_name` a separate field, or can it be derived from config?

Reply:
Nice catch, this is a duplicate.